recipe/meta.yaml

```
@@ -10,25 +10,27 @@ source:
   git_url:  https://github.com/pyg-team/pyg-lib/
   git_rev: {{ version }}
 
+  patches:
+    - fix-cmake.patch
+
 build:
-  number: 1
+  number: 10
```

Why 10? We're not counting in binary...
```suggestion
  number: 2
```